.docker/wiremock/fbs/mappings/fees-v2-without-parameters-ed922f92-46bb-46c6-926f-73678410fcd0.json

```diff
@@ -0,0 +1,23 @@
 \"periodical\": null,\n        \"materialGroup\": {\n          \"name\": \"standard\",\n          \"description\": \"31 dages lånetid til alm lånere\"\n        }\n      },\n      {\n        \"materialItemNumber\": \"5218297864\",\n        \"recordId\": \"45951685\",\n        \"periodical\": null,\n        \"materialGroup\": {\n          \"name\": \"standard\",\n          \"description\": \"31 dages lånetid til alm lånere\"\n        }\n      }\n    ]\n  },\n  {\n    \"feeId\": 434538,\n    \"type\": \"fee\",\n    \"reasonMessage\": \"Gebyr (for sent)\",\n    \"amount\": 120,\n    \"dueDate\": \"2022-05-08\",\n    \"creationDate\": \"2022-04-06\",\n    \"paidDate\": null,\n    \"payableByClient\": true,\n    \"materials\": [\n      {\n        \"materialItemNumber\": \"5136106556\",\n        \"recordId\": \"53067034\",\n        \"periodical\": null,\n        \"materialGroup\": {\n          \"name\": \"standard\",\n          \"description\": \"31 dages lånetid til alm lånere\"\n        }\n      },\n      {\n        \"materialItemNumber\": \"5232011841\",\n        \"recordId\": \"54058969\",\n        \"periodical\": null,\n        \"materialGroup\": {\n          \"name\": \"standard\",\n          \"description\": \"31 dages lånetid til alm lånere\"\n        }\n      }\n    ]\n  },\n  {\n    \"feeId\": 306404,\n    \"type\": \"fee\",\n    \"reasonMessage\": \"Gebyr (for sent)\",\n    \"amount\": 2.56,\n    \"dueDate\": \"2020-04-15\",\n    \"creationDate\": \"2019-10-18\",\n    \"paidDate\": null,\n    \"payableByClient\": true,\n    \"materials\": [\n      {\n        \"materialItemNumber\": \"3839631447\",\n        \"recordId\": \"26285283\",\n        \"periodical\": null,\n        \"materialGroup\": {\n          \"name\": \"standard\",\n          \"description\": \"31 dages lånetid til alm lånere\"\n        }\n      }\n    ]\n  },\n  {\n    \"feeId\": 377242,\n    \"type\": \"compensation\",\n    \"reasonMessage\": \"Erstatning (test)\",\n    \"amount\": 1,\n    \"dueDate\": \"2021-08-10\",\n    \"creationDate\": \"2021-02-11\",\n    \"paidDate\": null,\n    \"payableByClient\": true,\n    \"materials\": [\n      {\n        \"materialItemNumber\": \"271358741\",\n        \"recordId\": \"01484524\",\n        \"periodical\": null,\n        \"materialGroup\": {\n          \"name\": \"standard\",\n          \"description\": \"31 dages lånetid til alm lånere\"\n        }\n      }\n    ]\n  },\n  {\n    \"feeId\": 411199,\n    \"type\": \"compensation\",\n    \"reasonMessage\": \"Gebyr (for sent)\",\n    \"amount\": 1.5,\n    \"dueDate\": \"2021-12-24\",\n    \"creationDate\": \"2021-11-24\",\n    \"paidDate\": null,\n    \"payableByClient\": true,\n    \"materials\": [\n      {\n        \"materialItemNumber\": \"262088571\",\n        \"recordId\": \"06964206\",\n        \"periodical\": null,\n        \"materialGroup\": {\n          \"name\": \"standard\",\n          \"description\": \"31 dages lånetid til alm lånere\"\n        }\n      }\n    ]\n  }\n]",
```

Comment:
Don't we need dynamic dates in the response body?

Reply:
No, I don't think so. Why?

Reply:
The dates determines how the application works. I don't have an exact overview of the functionality/logic of the apps in this matter, but dates like `dueDate` and `creationDate` are fixed in the changes here. I might not be a problem, but if we want the dates in the data to be relative to current date in order to have the right scenario then we need dynamic dates.
Does it make sense?

Reply:
Uuu, yes yes yes, that makes total sense. I just don't see a reason to spend time on it, since the old request looks exactly the same. And I don't recall ever having problems with it.